WeatherPlannerDemo/Network/QueryParameters/CityQueryParameters.swift

```diff
@@ -0,0 +1,16 @@
+//
+//  CityQueryParameters.swift
+//  WeatherPlannerDemo
+//
+//  Created by Alen Sebalj on 07.12.2020..
+//
+
+import Foundation
+
+struct CityQueryParameters: QueryParameters {
+    let q: String
```

Comment:
Rename this to something more readable. Do you need to set some value to this property?